flaky/_flaky_plugin.py

```diff
@@ -94,6 +104,32 @@ def _handle_test_error_or_failure(self, test, err):
             self._log_test_failure(name, err, message)
             return False
 
+    def _should_rerun_test(self, test, name, err):
+        """
+        Whether or not a test should be rerun.
+        This is a pass-through to the test's rerun filter.
+
+        :param test:
+            The test that has raised an error
+        :type test:
+            :class:`nose.case.Test` or :class:`Function`
+        :param name:
+            The test name
+        :type name:
+            `unicode`
+        :param err:
+            Information about the test failure (from sys.exc_info())
+        :type err:
+            `tuple` of `class`, :class:`Exception`, `traceback`
```

Comment:
Why is this :class:`Exception` and not `Exception` (it is `traceback` and not :class:`traceback`)?